### ISSUE TITLE
Add support for ktlint baseline

### ISSUE THREAD

**Issue by shaobin0604** [JLLeitschuh/ktlint-gradle#414]
Ktlint https://github.com/pinterest/ktlint
> to add a baseline to check against prepend following args: "--baseline=ktlint-baseline.xml"

**Comment by JLLeitschuh** [comment]
As far as I'm aware, this is not a feature we currently support yet.

@Tapchicoma please confirm?

**Comment by Tapchicoma** [comment]
It will be available in `0.40.0` release. Eventually this plugin will also add such support.

**Comment by s1monw1** [comment]
> It will be available in 0.40.0 release. Eventually this plugin will also add such support.
This comment is misleading as this update already happened by now but the support for baselining isn't given yet, right? Can you give an update? Will it be included in the next update?

**Comment by Tapchicoma** [comment]
Adding baseline support is a separate task from updating default KtLint version to `0.40.0`. I've planned to work on it for the next release as `10.0.0` already has quite a lot of changes.

**Comment by firmwaves** [comment]
is there any workaround for this in the meantime?

**Comment by firmwaves** [comment]
@Tapchicoma any updates on this? or any ways for us to help out?

**Comment by Tapchicoma** [comment]
I will add baseline support after #459 PR will be merged (support require some changes from KtLint `0.41.0` release).
